Makefile

```diff
@@ -19,6 +19,7 @@ TRANSPORT_CMD_NAME ?= ocm-transport-controller
 IMAGE_TAG ?= $(shell git rev-parse --short HEAD)
 CONTROLLER_MANAGER_IMAGE ?= ${DOCKER_REGISTRY}/${CONTROLLER_MANAGER_CMD_NAME}:${IMAGE_TAG}
 TRANSPORT_IMAGE ?= ${DOCKER_REGISTRY}/${TRANSPORT_CMD_NAME}:${IMAGE_TAG}
+INSTALL_KUBEFLEX ?= true
```

Comment:
Why is this an option? Our pre-reqs.md doc says that KubeFlex is a pre-req for building and using KubeStellar.

Reply:
Since the test script create the kind cluster using `kflex init -c`, Kubeflex is already deployed in the cluster thus making necessary a flag to avoid trying to redeploy it with the `core-chart`.

Reply:
It's used in the context of the helm chart and it's trying to mimic the option in the `value.yaml`